File: C++/switch.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	setlocale(LC_ALL, "Russian");
    cout << "Введите число: ";
    int operand;
    cin >> operand;

    switch (operand)
    {
    case 5:
        cout << "5 = "  << operand << endl;
        break;
    case 10:
        cout << "10 = " << operand << endl;
        break;
    case -2:
        cout << "-2 = " << operand << endl;
        break;
    case -6:
        cout << "-6 = " << operand << endl;
        break;
    case 0:
        cout << "0 = " << operand << endl;
        break;
    default:
        cout << "Вы ошиблись" << endl;
        break;
    }

	system("pause");
}

File: JavaScript/switch.js


let operand = +prompt("Введите число");

switch (operand) {
    case 5:
        console.log("5 = " + operand);
        break;
    case 10:
        console.log("10 = " + operand);
        break;
    case -2:
        console.log("-2 = " + operand);
        break;
    case -6:
        console.log("-6 = " + operand);
        break;
    case 0.5:
        console.log("0.5 = " + operand);
        break;
    default:
        console.log('Вы ошиблись');
}